### ISSUE TITLE
[ADR]: Method & technology for collecting targeted feedback 

### ISSUE THREAD

**Issue by andycochran** [HHS/simpler-grants-gov#306]
### Description

**tl;dr** = We need a survey tool 

This ADR is intended to determine a method and technology for collecting feedback through the Simpler.Grants.gov website. We are interested in hearing from all archetypes, but especially the Novice (first-time applicants or users who've created a workspace but haven't applied yet). But anyone should be able to respond regardless of their experience with federal grants. 

~Feedback should be collected through open-ended text boxes that allow users to write whatever they want. Direct questions can be address or ignored.~ With PRA general clearance having been obtained, it's no longer required that responses be open-ended, as survey approval can now be fast-tracked. 

This should consider the "contact us at [email]" CTA, and how/whether that may be a different use case. Allowing users to submit questions or comments privately (as an alternative to a public forum or listserv) may be a particular way an instance of this tool is used, but this is not the sole function. There is a separate ADR for "contact us" (#682). 

**Example use case:** We have launched a new set of search filters and want to send a survey to specific communities to get their feedback. 

### DACI

| role | people |
|--------|--------|
| Driver | @andycochran |
| Approver(s) | @lucasmbrown-usds, @itsemilyianacone |
| Contributor | @sumiat, @EOKENAVA, Adriana Weitzman |
| Informed | @acouch, @daphnegold, @SammySteiner, @chouinar, @widal001, @sarahknoppA6 | 

### Potential options

- [Ethnio](https://ethn.io/)
- [Touchpoints](https://touchpoints.digital.gov/)
- Microsoft Form (created by HHS's 365) 
- HTML form that posts to HHS's Sharepoint
- HTML form that creates a GitHub issue, letting users know their response will create a public issue w/o needing to create a GitHub account
- Create and embed an open text form with a user research tool like [Dovetail](https://dovetail.com/), so that feedback  is captured, centralized, and analyzed in the same place as future research studies
- Google form (link to, embed, or post to)
- Ask that folx use the Google Group openly

### Decision criteria

- Should not feel like a black box; user understands how their submission will be used
- Has a better UX than sending an email 
- Data is stored in a structured way 
- Metadata (tags, fields, etc) make submissions easy to sort through, analyze, and synthesize

### Definition of Done

- [ ] The approvers for this decision have been identified (ideally before work on the ADR starts)
- [ ] The ADR is created and stored in `documentation/decisions/adr`
- [ ] The ADR has been reviewed and approved by the approvers listed above
- [ ] The ADR satisfies requirements that are outlined in the ADR template
- [ ] Any follow-up tickets have been created (if necessary)

### Open questions / Notes
- What are the use cases? How will we be using the data? 
- Design choices: Will we collect PII? Will we respond to the respondents? 
- Are there federal data retention requirements for surveys that should be considered (e.g. where the data is stored, how long it must be kept)?

**Comment by andycochran** [comment]
Chatted yesterday with @itsemilyianacone, who will look into whether HHS has a [Qualtrics](https://www.qualtrics.com/) license that can be used for the beta site.

**Comment by sarahknoppA6** [comment]
Could we find out if this is done or still in progress?

**Comment by andycochran** [comment]
ADR for #691 prescribes using Ethnio for design research. It includes general surveys feature that satisfies the criteria listed in this issue. I think this issue (in addition to #117) can be closed by #691.

**Comment by margaretspring** [comment]
@andycochran is this still relevant - please check out & close if appropriate.

**Comment by widal001** [comment]
Beep boop: Automatically setting the point and sprint values for this issue in project HHS/13 because they were unset when the issue was closed.
